src/main/java/nextstep/qna/domain/Question.java

```diff
@@ -65,14 +66,33 @@ public NsUser getWriter() {
 
     public void addAnswer(Answer answer) {
         answer.toQuestion(this);
-        answers.add(answer);
+        answers.addAnswer(answer);
     }
 
-    public boolean isOwner(NsUser loginUser) {
+    public List<DeleteHistory> delete(NsUser loginUser) throws CannotDeleteException {
+        List<DeleteHistory> deleteHistories = deleteAnswers(loginUser);
```

Comment:
```suggestion
        List<DeleteHistory> deleteHistories = answers.delete(loginUser);
```

굳이 한 라인인데 메서드로 분리할 필요가 있을까?